### ISSUE TITLE
ABL not working

### ISSUE THREAD

**Issue by dreschpe** [TheAntTeam/TheAntFarm#27]
I have copied the source back into the main dir to get the project running.
 
![test](https://user-images.githubusercontent.com/6898010/184431981-3444003d-4657-4b80-a7bf-e030c93dcf01.jpg)

Here I have tried three times to mill a small PCB. The ABL is not working...
Then I used bCNC with the Ant Farm G Code.  This time it works. ABL with bCNC. 
I have started the ABL process and have it activated.

Peter

**Comment by TheAntTeam** [comment]
@dreschpe, it seems that there is an issue with the ABL on the bottom side that we are investigating. Are you trying to mill a bottom side?

**Comment by dreschpe** [comment]
no ist was a single side front.Am 19.08.2022 09:43 schrieb TheAntTeam ***@***.***>:
@dreschpe, it seems that there is an issue with the ABL on the bottom side that we are investigating. Are you trying to mill a bottom side?

**Comment by bvarner** [comment]
I had the same issues on the top side as well.

I started using bCNC and things work fine.

**Comment by TheAntTeam** [comment]
Yes, it is possible. There is a bug that prevents ABL information from loading correctly. The bug can show up on both sides. We are trying to solve.

**Comment by TheAntTeam** [comment]
Hi, can you try the fix? I have done some tests and It seems working

**Comment by dreschpe** [comment]
I have pulled the actual code and it is working for a top layer.

Best Regards,
Peter
